### ISSUE TITLE
Morning mode

### ISSUE THREAD

**Issue by miguelpruivo** [Roaders/node-red-contrib-soma-smart-shade#1]
Does this work with Soma Smart Shades 2 and do you think that the morning (slow) mode can be implemented? Where did you get the commands from? Did you use a proxy to intercept them or are just public available?

Thank you!

Edit: just found it, looks like [there is](https://support.somasmarthome.com/hc/en-us/articles/360026064234-HTTP-API). I'll implement it and issue a PR.

**Comment by Roaders** [comment]
yes, just found the same thing on the api that you have linked to. Looks like it will be pretty trivial to add

**Comment by miguelpruivo** [comment]
How does your plugin differ from [this one](https://github.com/dionmes/node-red-contrib-soma-smartblinds2)? From the developer, the second doesn't need Soma Connect installed, does this mean that yours do?

Also, could you publish the plugin to the NPM so it's exposed into the NR plugins?

Thanks!

**Comment by Roaders** [comment]
mine calls apis on the soma-connect box. The other one makes direct bluetooth requests to the blinds.

**Comment by Roaders** [comment]
it is on npm
https://www.npmjs.com/package/node-red-contrib-soma-smart-shade

**Comment by miguelpruivo** [comment]
I don't have the Soma Connect Box, only NR. Does that mean I can't use yours?

**Comment by Roaders** [comment]
yes but you don't need to buy the soma connect. You can just use a raspberry pi near the blinds to control it. I used a raspberry pi nano and it worked as well as any bluetooth control does (which is not flawlessly). if your raspberry pi that runs node red is right next to the blinds then the other might work ok. If it's in a different room I would expect you to have even more issues than I had.

I no longer use soma blinds as in my experience bluetooth connections are not reliable. I now use zigbee devices.
